src/lib/RpcApi.ts

```diff
@@ -103,8 +149,6 @@ export default class RpcApi {
             if (window.opener === null && window.history.length > 1 && !window.location.hash) {
                 location.href = Config.redirectTarget;
             } else {
-                // Not using routerReplace here as that method exports the state which does not exist when there is no
-                // request.
                 this._router.replace(`/${REQUEST_ERROR}`);
```

Comment:
For this call I now get the following Error:
```
RpcApi.ts:218 Uncaught (in promise) TypeError: Cannot read property 'reply' of undefined
    at RpcApi._reply (RpcApi.ts:218)
    at RpcApi.reject (RpcApi.ts:190)
    at RpcApi._router.beforeEach (RpcApi.ts:110)
    at iterator (vue-router.esm.js:2154)
    at step (vue-router.esm.js:1880)
    at runQueue (vue-router.esm.js:1888)
    at HTML5History.confirmTransition (vue-router.esm.js:2181)
    at HTML5History.transitionTo (vue-router.esm.js:2068)
    at HTML5History.replace (vue-router.esm.js:2411)
    at eval (vue-router.esm.js:2824)
```

Reply:
Should be fixed now, please take a look.